### DISCUSSION TITLE
Redirect a request coming from authorized user

### DISCUSSION THREAD

**Discussion by amr3k** [fastapi-users/fastapi-users#527] [Q&A]
Hello, I hope you all are doing well.

I'm using jwt based authentication and I wonder if the app got `GET /login`, Is it possible to check if that request is coming from logged-in user and redirect to e.g. `/`?
I'm already using exception-based-redirection following #423 where any unauthorized request gets redirected to `/login`. Now I'd love to be able to do the opposite as well.

Thanks.

**Comment by frankie567** [comment]
I don't think that's possible without reimplementing the `/login` route yourself.

We've designed the library to be used as a REST API, so a redirect pattern is not very relevant here.
